sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ResolverClientOptions.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.ModelsRepository
+{
+    /// <summary>
+    /// Options that allow configuration of requests sent to the ModelRepositoryService.
+    /// </summary>
+    public class ResolverClientOptions : ClientOptions
+    {
+        internal const ServiceVersion LatestVersion = ServiceVersion.V2021_02_11;
+
+        /// <summary>
+        /// The versions of Azure Digital Twins supported by this client
+        /// library.
+        /// </summary>
+        public enum ServiceVersion
```

Comment:
This can be ignored for now, we need to have a discussion about this since there is no backing service with different API versions. 
Will bring it up during the meeting with SDK rep